### ISSUE TITLE
How to parse the caffe-generated LMDB datasets with tf.LMDBReader

### ISSUE THREAD

**Issue by asjmasjm** [tensorflow/tensorflow#22892]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
PC
- **TensorFlow installed from (source or binary)**:
Binary
- **TensorFlow version (use command below)**:
1.9.0
- **Python version**:
2.7.15
- **Bazel version (if compiling from source)**:
Not compiled from source
- **GCC/Compiler version (if compiling from source)**:
Not compiled from source
- **CUDA/cuDNN version**:
CUDA 9.0
- **GPU model and memory**:
GTX 1060 6G

### Describe the problem
After importation through the tf.LMDBReader(), we are able to get the key and value of the .mdb. But we cannot further parse the value through the tf.parse_single_example API as no sub-keys like "image_raw" or "label" existed in the Caffe-generated .mdb dataset. As in Caffe, the properties of the image or the label data are available with this data block converted to Caffe-defined Datum structure and we can get the width through methods like Datum.width(). So what we should do in tensorflow?

### Source code / logs
filename_queue = tf.train.string_input_producer(['path_to_the_data.mdb'], num_epochs=None)
reader = tf.LMDBReader()
cur_key_val, serialized_example = reader.read(filename_queue)
features = tf.parse_single_example(
        serialized_example,
        features={
             '?????': tf.VarLenFeature(tf.string),
             '?????': tf.VarLenFeature(tf.string)
         })
label = features['?????']
image = features['?????']

**Comment by ppwwyyxx** [comment]
No you can not parse caffe LMDB with `tf.LMDBReader`.
In fact, using caffe's lmdb file is a bad idea because how inefficient it is: it stores image as arrays instead of jpegs, and can make data loading sometimes [many times slower](https://github.com/tensorpack/tensorpack/issues/124#issuecomment-277206505)

**Comment by asjmasjm** [comment]
> No you can not parse caffe LMDB with `tf.LMDBReader`.
> In fact, using caffe's lmdb file is a bad idea because how inefficient it is: it stores image as arrays instead of jpegs, and can make data loading sometimes [many times slower](https://github.com/tensorpack/tensorpack/issues/124#issuecomment-277206505)

Thank you for your response and we have turned to the TFRecord method to feed the data instead.

**Comment by harshini-gadige** [comment]
@asjmasjm Hi, looks like your question is answered. Feel free to close this issue if you do not have any outstanding issues related to this.

**Comment by asjmasjm** [comment]
Thanks @harshini-gadige and close this topic.
